ComputerPlayer.js

```diff
@@ -0,0 +1,39 @@
+const { dictionary } = require("./Dictionary.js")
+
+  class ComputerPlayer {
+    constructor (name = "Computer") {
+      this.dictionary = dictionary;
+      this.name = name;
+      this.userGuess = new guess()
```

Comment:
this is declaring a new instance of a class called guess, which doesn't exist...